middleware/http/wasm/benchmark_test.go

```diff
@@ -28,19 +28,19 @@ func BenchmarkNative(b *testing.B) {
 }
 
 func BenchmarkTinygo(b *testing.B) {
-	path := "./internal/e2e-guests/rewrite/main.wasm"
+	path := "file://internal/e2e-guests/rewrite/main.wasm"
```

Comment:
nit: if renaming we should also rename the var to url

Reply:
Thanks for your review, done.